### DISCUSSION TITLE
Suchfelder - Beispiel für die Eingabe

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#61] [Q&A]
Hallo Jürgen!

Gibt es eine Möglichkeit bei der Suche ein "Beispiel für die Eingabe" anzeigen zu lassen?
Im CMS wäre dafür ein Feld vorgesehen, allerdings sehe ich bei der Suche nirgends einen Hinweis dafür (verwende die aktuelle Version 5.23.1601). 
 
![image](https://user-images.githubusercontent.com/122986367/234857768-d3105216-469c-4ea4-a9a1-7b4bb6e08b12.png)

Konkreter Fall:
Im Atlas kann nach Straßenkilometern gesucht werden --> https://atlas.noe.gv.at/atlas/portal/noe-atlas/map/Verkehr/NÖ%20Straßen

Straße: A2; Kilometer: 10,2 --> OK (Der Kilometerwert in der Suche ist eine Zahl mit einer Nachkommastelle und damit kommt ein Ergebnis) --> https://atlas.noe.gv.at/atlas/portal/noe-atlas/map/Verkehr/N%C3%96%20Stra%C3%9Fen?abfragethema=strassenhektometer&such1=A2&such2=10,2

Straße: A2; Kilometer: 10,25 --> Nicht OK (Der Kilometer hat zwei Nachkommastellen, deswegen gibt es kein Ergebnis) --> https://atlas.noe.gv.at/atlas/portal/noe-atlas/map/Verkehr/N%C3%96%20Stra%C3%9Fen?abfragethema=strassenhektometer&such1=A2&such2=10,25

Es ist zwar möglich eine Regex zu hinterlegen, allerdings gibt die Meldung nur aus, dass der Wert nicht gültig ist, aber nicht warum.

![image](https://user-images.githubusercontent.com/122986367/234859247-a56c5bc6-f80f-4a39-8d05-1d228d06e653.png)

Wie kann ich dem User mitgeben, in welcher Form eine Sucheingabe vorzunehmen ist, die ein gültiges Ergebnis zurückgibt?

Kann dafür das Feld "Beispiele für die Eingabe" verwendet werden oder ist das ein Relikt aus WebGIS4?
Wäre es ev. möglich (analog wie im cms) ein Fragezeichen anzubieten, wo mit Klick darauf ein Hinweistext aufklappt?
Könnte man ev. die Regex erweitern, dass dort ein Hinweis mitgeliefert wird, wie eine korrekte Eingabe ist?
Könnte ev. auch ein Beispiel im Textfeld eingetragen sein (siehe z.B. die Google-Suche "Mit Google suchen oder eine URL eingeben")?

Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Stimmt, das kommt noch aus der WebGIS 4 Ära. 
Ein einfaches Beispiel für die Eingabe ist allerdings schon sinnvoll. Der Wert, den du hier einträgst wird in Zukunft im Eingabefeld als *Placeholder* angezeigt (grauer Text, der angezeigt wird, bis der Anwender selber etwas eingibt):

![image](https://user-images.githubusercontent.com/62870568/235601699-b855a747-0187-42e3-928e-106a460d4c1c.png)

*Placeholder* sind eigentlich State of the Art. Falls man hier längere Texten anbieten möchte, müssen wir uns vielleicht einmal etwas anderes überlegen.
